src/Request.php

```diff
@@ -626,58 +626,58 @@ public function execute() {
 			}, $this->params);
 		}
 
-		if ( $this->params instanceof OperationParams ) {
+		// If $this->params isnt an array or an OperationParams instance, then something probably went wrong.
+		if ( ! $this->params instanceof OperationParams ) {
```

Comment:
The PHPStan-requested change here is that `execute()` doesn't return anything if `$this->params` is neither an array or an `OperationParams` instance.

This is solved by "returning early" instead of wrapping the logic in a conditional.

I _believe_ throwing an exception is the right approach here, but because of the method recursion and the lack of `array` shape for `$this->params`/`$this->data`, this could use a second set of eyes.

cc: @jasonbahl